docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md

```diff
@@ -0,0 +1,85 @@
+
+| localization mode | method  | map type        |
+| ----------------- | ------- | --------------- |
+| LiDAR-based       | NDT     | point cloud map |
+| camera-based      | YabLoc  | vector map      |
+| GNSS/IMU-based    | Eagleye | -               |
+
+## LiDAR-based localizer (default)
+
+By default, Autoware launches [ndt_scan_matcher](https://github.com/autowarefoundation/autoware.universe/tree/main/localization/ndt_scan_matcher) for localization.
+
+## Camera-based localizer
+
+You can use YabLoc as a camera-based localization method. For more details on YabLoc, please refer to the [YabLoc Guide](yabloc-guide.md).
+
+To use YabLoc as a pose_estimator, add `localization_mode:=camera` when launching Autoware.
+By default, the `localization_mode` is set to `lidar`.
+By specifying this command-line argument, YabLoc nodes will be automatically launched while the NDT nodes will not be started.
+
+Here is an example of a launch command:
+
+```bash
+ros2 launch autoware_launch autoware.launch.xml \
+  vehicle_model:=YOUR_VEHICLE \
+  sensor_kit:=YOUR_SENSOR_KIT map_path:=/PATH/TO/YOUR/MAP \
+  localization_mode:=camera # Add this argument
+```
+
+## GNSS/IMU-based localizer
+
+You can use Eagleye as a GNSS/IMU-based localization method. For more details on Eagleye, please refer to the [Eagleye Guide](eagleye-guide.md).
+
+Eagleye has a function for position estimation and twist estimation, namely `pose_estimator` and `twist_estimator`, respectively.
+When running Eagleye in twist_estimator mode, ndt_scan_matcher is used as the pose_estimator.
+Eagleye will improve scan matching by providing accurate twists using GNSS doppler.
+
+To use Eagleye, it requires both specifying the command-line arguments and modifying the launch file.
+
+### 1. Modifying Autoware launch files
+
+When using Eagleye, comment out `tier4_localization_component.launch.xml` and start `map4_localization_component.launch.xml` in [`autoware.launch.xml`](https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/autoware.launch.xml).
+Please refer to the following snippet for the modification details:
+
+```xml
+  <!-- Localization -->
+  <group if="$(var launch_localization)">
+    <!-- <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_localization_component.launch.xml"/> -->
+    <include file="$(find-pkg-share autoware_launch)/launch/components/map4_localization_component.launch.xml"/>
+  </group>
+```
+
+NOTE: Please refer to [`map4_localization_launch`](https://github.com/autowarefoundation/autoware.universe/tree/main/launch/map4_localization_launch) in the `autoware.universe` package and [`map4_localization_component.launch.xml`](https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/components/map4_localization_component.launch.xml) in `autoware_launch` package for information on how to modify the localization launch.
+
+### 2. Execution command
```

Comment:
Same as above

Reply:
I merged the short sub-sub-sections into sub-sections in 50ff60b